html/modules/custom/docstore/src/ParseQueryParameters.php

```diff
@@ -213,7 +213,13 @@ protected function buildConditionGroup(array $members, string $conjunction, Quer
         $conditions->addConditionGroup($subgroup);
       }
       else {
-        $conditions->addCondition($filter['condition']['path'], $filter['condition']['value'], $filter['condition']['operator']);
+        // Handle multiple values for IN.
+        if ($filter['condition']['operator'] === 'IN' || $filter['condition']['operator'] === 'NOT IN') {
+          $conditions->addCondition($filter['condition']['path'], explode(',', $filter['condition']['value']), $filter['condition']['operator']);
```

Comment:
That seems to deviate from the way the JSON API handles multiple values: https://www.drupal.org/docs/core-modules-and-themes/core-modules/jsonapi-module/filtering#s-4-filtering-with-arrays-get-nodes-created-by-users-admin-john.

Using a comma to separate values would be an issue when trying to filter by terms that contain commas like some reliefweb disasters.

Reply:
External entities is sending it like this, hence the change. Changing it in external entities will be more complicated I assume.

I propose we leave it for now until we can solve it upstream

Reply:
I can handle this in the ExternalEntityStorageClient